.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of the issue in the Title above -->
```

Comment:
It's intended. They will help the user (that will read the HTML comment) and they will not render on the Issue/PR so other people won't see them (like the description of this PR #44 )